Mergin/utils.py

```diff
@@ -1505,3 +1505,12 @@ def get_layer_by_path(path):
         safe_file_path = layer_path.split("|")
         if safe_file_path[0] == path:
             return layer
+
+def bytes_to_mb(bytes : int):
+    """
+    Convert bytes to megabytes assuming base of bytes in binary
+    
+    Returns:
+    float: Result in megabytes
+    """
+    return bytes / (2**20)
```

Comment:
Thinking about this again, it will be nicer if it could convert the input size into human readable output. Imagine you are on a team plan (with 50GBs), you would now see a max limit of 50000 MB, but we could show 50 GB instead to make it more readable.

There is a function like this in our mobile app already. Can you please add an equivalent of it?
https://github.com/MerginMaps/mobile/blob/052b5b663e3ad6095995f58adb16f40a82c302ae/core/coreutils.cpp#L314-L323

In fact, moving this function to `python-api-client` will be a good idea so that we can reuse it later in other places too.